code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -32,7 +31,8 @@
 
 	create_internal_organs()
 
-	ADD_TRAIT(src, TRAIT_NEVER_WOUNDED, ROUNDSTART_TRAIT)
+	ADD_TRAIT(src, TRAIT_NEVER_WOUNDED, INNATE_TRAIT)
```

Comment:
This isn't declared in the PR body, 99% of the traits given to mobs just for existing come from the INNATE_TRAIT source, rather than ROUNDSTART_TRAIT.